### ISSUE TITLE
Retrieving monitoring status for Icinga currently not working

### ISSUE THREAD

**Issue by stdevel** [stdevel/katprep#64]
```
INFO:katprep_maintenance:No downtime for host 'pinkepank.giertz.loc' found, probably cleaned-up.
Traceback (most recent call last):
  File "/home/cstan/.local/bin/katprep_maintenance", line 9, in <module>
    load_entry_point('katprep==0.1', 'console_scripts', 'katprep_maintenance')()
  File "/home/cstan/katprep/katprep/maintenance.py", line 726, in cli
    main(options, args)
  File "/home/cstan/katprep/katprep/maintenance.py", line 705, in main
    options.func(options, options.func)
  File "/home/cstan/katprep/katprep/maintenance.py", line 370, in verify
    crit_services = MON_CLIENTS[get_host_param_from_report(REPORT, host, "katprep_mon")].get_services(mon_name)
  File "/home/cstan/katprep/katprep/clients/NagiosCGIClient.py", line 427, in get_services
    result = self.__api_get(url)
  File "/home/cstan/katprep/katprep/clients/NagiosCGIClient.py", line 175, in __api_get
    return self.__api_request("get", sub_url)
  File "/home/cstan/katprep/katprep/clients/NagiosCGIClient.py", line 146, in __api_request
    raise SessionException("Unable to authenticate")
katprep.clients.SessionException: Unable to authenticate
```

**Comment by stdevel** [comment]
Reprocuded this issue on an Icinga 1.14.0 installation with multiple hundred hosts on EL6. Not able to reproduce the issue in my lab with Icinga 1.14.0 on EL7.

**Comment by stdevel** [comment]
Found another brilliant web scraping issue with Nagios. 🙄

**Comment by stdevel** [comment]
Fixed in commit c1db8b96db0c94a17290dbff3c223b681e0541c8.
